ext/flash/socket.rs

```diff
@@ -26,6 +29,7 @@ pub struct Stream {
   pub parse_done: ParseStatus,
   pub buffer: UnsafeCell<Vec<u8>>,
   pub read_lock: Arc<Mutex<()>>,
+  pub _pinned: PhantomPinned,
```

Comment:
This is not directly related to this PR, but I feel this field is a must so we can make sure that `Stream` is not moved when it's `Pin`ned.